stacker/exceptions.py

```diff
@@ -128,3 +128,13 @@ def __init__(self, variable, validator, value, exception=None):
 
     def __str__(self):
         return self.message
+
+
+class UnallowedValue(Exception):
+    """Exception raised when the value provided to a variable is not allowed.
+    """
+    def __init__(self, variable, value, allowed_values, *args, **kwargs):
```

Comment:
This might be hard to find, since it doesn't list what blueprint it's a part of.  I actually am starting to feel like we should be doing more with catching exceptions at the blueprint, plan or stack level to provide more context, rather than needing to pass that context down to each exception in order to figure out where things are breaking.